### PR TITLE
44478-B: Update README.md

### PR DESCRIPTION
**Pull request by jonquil2002**
SC: https://app.shortcut.com/replicated/story/44478/replicated-docs-readme-add-guidance-for-folders-and-sidebar-js

Some corrections to the table formatting and other tweaks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,9 +24,9 @@ Run this command to install the `yarn` CLI:
 $ yarn
 ```
 
-### Create a Local Development Server and Preview
+### Create a Local Development Server and WSYWYG Preview
```
Comment (paigecalvert) [inline README.md:27]: Should this be WYSIWYG? (What You See Is What You Get)?

I didn't see instances of WSYWYG on a quick google search

File: README.md [hunk 2]
```diff
@@ -24,9 +24,9 @@ Run this command to install the `yarn` CLI:
...
 
-Run the following command to start a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+Run the following command to start a local development server and opens up a browser window. Most changes are reflected live without having to restart the server. This preview shows the formatting and styles as they would render on the live site (also known as WSYWYG), which can be more helpful than viewing content in a VS Code preview.
```
Comment (paigecalvert) [inline README.md:29]: Typo:
>Run the following command to start a local development server **and opens** up

**Review by paigecalvert — DISMISSED** [review]
Just a couple notes!

File: README.md (hunk 1 above)
Comment (jonquil2002) [reply README.md:27]: You are correct - I am SO bad at acronyms! I'll fix it.

Comment (paigecalvert) [reply README.md:27]: To be fair this is a way-too-long acronym!!

**Review by paigecalvert — APPROVED** [review]

**Pull request merged**